Exercises_7_a_b_c.cpp

```diff
@@ -0,0 +1,98 @@
+
+#include "stdafx.h"
+using namespace cv;
+using namespace std;
+
+
```

Comment:
In all files, I want the program to say what it is:
void help(const char **argv) {
  cout << "\n\n"
       << "Exercises 7a-c:\n"
       << "Call:\n" << argv[0]
       << endl;
}